.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,88 @@
+name: Bug Report
+description: File a bug report
+title: "[BUG]: "
+labels: ["bug", "triage"]
+projects: ["Team Engineering Project"]
```

Comment:
The `projects` field does not seem to be allowed in the YAML schema for issue templates.

```suggestion
```